budget app backend/modules/jwt.js

```diff
@@ -2,13 +2,15 @@ const jwt = require("jsonwebtoken");
 const { JWT_SECRET } = require("../config");
 
 module.exports.create_token = function (payload) {
-    return jwt.sign(payload, JWT_SECRET)
+    return "Bearer " + jwt.sign(payload, JWT_SECRET)
```

Comment:
I included passport-jwt 's bearer token verification. So POSTMAN should be updated to take into account that each route is protected with authorization header.

Reply:
Token shouldnt have prefix "Bearer ". 
https://jwt.io/introduction
FE should add "Bearer "

Reply:
> Token shouldnt have prefix "Bearer ". https://jwt.io/introduction FE should add "Bearer "

It's what our teacher showed us. But I'll take note of that, thanks